configure.win

```diff
@@ -163,8 +65,8 @@ fi # end check on "${MSBUILD_EXE}"
 # Note: == seems not supported in sh (ash??)
 if [ "$VS_COMNTOOLS" = "" ]
```

Comment:
here VS_COMNTOOLS cannot be empty **per construction**. It's either set by user via VSCOMMON variable or is set above in the script to "C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/Common7"

I would rather check if the directory exists

Reply:
Ok done